Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Jira_Notification.yml

```diff
@@ -0,0 +1,289 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: f25546a1-47be-4b79-803e-013a06424a83
+    type: playbook
+    task:
+      id: f25546a1-47be-4b79-803e-013a06424a83
+      version: -1
+      name: Cortex ASM - Remediation Guidance
+      description: This playbook pulls remediation guidance off of a list based on ASM RuleID to be used in service owner notifications (email or ticketing system).
+      playbookName: Cortex ASM - Remediation Guidance
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      IssueTypeID:
+        complex:
+          root: alert
+          accessor: asmattacksurfaceruleid
+          transformers:
+          - operator: StripChars
+            args:
+              chars:
+                value:
+                  simple: '[\"]'
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 6e358c93-d64a-421a-8191-780a655d3739
+    type: regular
+    task:
+      id: 6e358c93-d64a-421a-8191-780a655d3739
+      version: -1
+      name: Set notification grid field
+      description: |-
+        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. For example:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      scriptName: GridFieldSetup
+      type: regular
+      iscommand: false
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      gridfield:
+        simple: asmnotification
+      keys:
+        simple: Type,Value,URL,Timestamp
+      val1:
+        simple: Jira
+      val2:
+        complex:
+          root: Ticket
+          accessor: Key
+      val3:
+        complex:
+          root: Ticket
+          accessor: Key
+          transformers:
+          - operator: FirstArrayElement
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: browse/
+              suffix: {}
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: JiraDomain.url
+                iscontext: true
+              suffix:
+                iscontext: true
+      val4:
+        simple: TIMESTAMP
```

Comment:
Observed this in few other places. Is TIMESTAMP a string? I'm guessing it should get the ISO time format and in the playbook run, I saw that the value is literal TIMESTAMP instead of an actual timestamp.